internal/core/workers/operation_execution_worker/operation_execution_worker_test.go

```diff
@@ -153,7 +153,7 @@ func TestSchedulerOperationsExecutionLoop(t *testing.T) {
 
 		operationLeaseStorage.EXPECT().GrantLease(gomock.Any(), expectedOperation.SchedulerName, expectedOperation.ID, config.OperationLeaseTtl).Return(nil)
 		operationLeaseStorage.EXPECT().RevokeLease(gomock.Any(), expectedOperation.SchedulerName, expectedOperation.ID).Return(nil)
-		operationLeaseStorage.EXPECT().RenewLease(gomock.Any(), expectedOperation.SchedulerName, expectedOperation.ID, config.OperationLeaseTtl).Return(nil)
+		operationLeaseStorage.EXPECT().RenewLease(gomock.Any(), expectedOperation.SchedulerName, expectedOperation.ID, config.OperationLeaseTtl).Return(nil).MinTimes(0)
```

Comment:
@caio why did we change this expectation to min times 0?

Reply:
Because if the operation runs very fast (which was the case here sometimes) it didn't have to renew lease any time